clippy_lints/Cargo.toml

```
@@ -23,6 +23,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0", optional = true }
 toml = "0.5.3"
 unicode-normalization = "0.1"
+unicode-script = "0.5"
```

`rustc` uses this as a transitive dep from `unicode-security` which depends on this with 

```suggestion
unicode-script = { version = "0.5.1", default-features = false }
```